src/mongo_types.ts

```diff
@@ -474,6 +474,38 @@ export class TypedEventEmitter<Events extends EventsDescription> extends EventEm
 /** @public */
 export class CancellationToken extends TypedEventEmitter<{ cancel(): void }> {}
 
+/** @public */
+export type Abortable = {
```

Comment:
(optional)

```suggestion
export type AbortOptions = {
```

Only because something like `FindOptions & AbortOptions` makes more sense than `FindOptions & Abortable` imo

Reply:
Why're we adding our own abortable, when we can just use the interface in `@types/node`?